src/components/screens/SendScreen/screens/TransactionAmountScreen.tsx

```diff
@@ -417,7 +417,7 @@ const TransactionAmountScreen: React.FC<TransactionAmountScreenProps> = ({
     resetTransaction();
 
     // Mark history to refresh when user navigates to history screen
-    markHistoryForRefreshAfterTransaction();
+    useHistoryStore.getState().markForRefreshAfterNavigation();
```

Comment:
why not extracting it at the beginning of the component?

`const { markForRefreshAfterNavigation } = useHistoryStore();`